Ladybird/Tab.cpp

```diff
@@ -30,6 +32,15 @@ Tab::Tab(BrowserWindow* window, StringView webdriver_content_ipc_path, WebView::
 
     m_view = new WebContentView(webdriver_content_ipc_path, enable_callgrind_profiling);
     m_toolbar = new QToolBar(this);
+    m_bookmarks_bar = new QListView(this);
+    m_bookmarks_bar->setFlow(QListView::LeftToRight);
```

Comment:
Feels a little odd each tab has its own bookmark bar, but maybe okay :)

Reply:
Yes, but the model is just one... Right now for every tab all widget are "duplicated"